src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
-body {
+$stopwatch-size: 80vmin;
+$circle-size: 5vmin;
+$minute-hand-length: 20vmin;
+$minute-hand-width: 3vmin;
+$second-hand-length: 38vmin;
+$second-hand-width: 1.5vmin;
+
+html {
   margin: 0;
+  padding: 0;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+  border-radius: 50%;
+  border: 1vmin dotted #000;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__circle {
+    position: absolute;
+    z-index: 1;
+    width: $circle-size;
+    height: $circle-size;
+    background-color: #f6a603;
+    border-radius: 50%;
+  }
+
+  &__minute-hand {
+    position: absolute;
+    width: $minute-hand-width;
+    height: $minute-hand-length;
+    background-color: #0700ff;
+    bottom: 50%;
+    transform-origin: center bottom;
+    animation: rotate 3600s steps(60) infinite;
+  }
+
+  &__second-hand {
+    position: absolute;
+    width: $second-hand-width;
+    height: $second-hand-length;
+    background-color: #2c8000;
+    bottom: 50%;
+    transform-origin: center bottom;
+    animation: rotate 60s linear infinite;
+  }
+
+  @keyframes rotate {
+    from {
+      transform: rotate(0deg);
+    }
+    to {
+      transform: rotate(360deg);
+    }
+  }
+}
+
+.stopwatch--speed-up {
+  .stopwatch__seconds-hand {
```

Comment:
The class name `.stopwatch__seconds-hand` seems to be incorrect. It should be `.stopwatch__second-hand` to match the class defined earlier in the file.